### ISSUE TITLE
Method '_GETset' not found in string|\TYPO3\CMS\Core\Utility\GeneralUtility

### ISSUE THREAD

**Issue by darthnorman** [DMKEBUSINESSGMBH/typo3-mktools#20]
TYPO3: 10.4
mktools: 10.0.5

In Classes/Frontend/Request/Parameters.php the method _GETset is depricated and was removed in TYPO3 v10.

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/9.5/Deprecation-86389-GeneralUtility_GETsetAndTSFE-mergingWithGetVars.html

This breaks the menu processor functionality of mktools (Classes/Utility/Menu/Processor/TranslatedRecords.php)

**Comment by darthnorman** [comment]
Maybe the method I was refering to, is not the real problem.
The problem is that mktools creates a link to a translation that doesn't exist. 

We use this in our language menu to see if news and courses have a translation we can link to:

```itemArrayProcFunc = DMK\Mktools\Utility\Menu\Processor\TranslatedRecords->process
            itemArrayProcFunc {
                menuConfiguration < lib.nav.language
                parametersConfiguration {
                    tx_news_pi1.news = tx_news_domain_model_news
                    tx_t3study_list.course  = tx_t3study_domain_model_course
                }
            }```

But even if there is no translation, the link is generated, resulting in a 404.

**Comment by hannesbochmann** [comment]
The translation utility got fixed in TYPO3 10.4 with commit https://github.com/DMKEBUSINESSGMBH/typo3-mktools/commit/5d14e91a81ed907cdac7ea2453eae1a1e4de5add
